.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -9,26 +9,26 @@ assignees: ''
 
 ## Bug Report
 
-<!-- Check this if you would like to implement a PR, we are more than happy to help you go through the process !-->
+<!-- If you would like to implement a PR, we are more than happy to help you go through the process!-->
 - [ ] I would like to work on a fix!
 
 <!--
 @babel/eslint-parser:
   If you are having issues with JSX you might want to check out eslint-plugin-react. If there's an issue with new experimental syntax you might need check if it's supported by @babel/eslint-plugin.
 -->
 
-**Current Behavior**
+**Current behavior**
 A clear and concise description of the behavior.
 
-- [REPL](babeljs.io/repl), [Codesandbox](https://codesandbox.io/s/babel-repl-custom-plugin-7s08o?file=/src/index.js), or GitHub Repo link if applicable:
+- [REPL](babeljs.io/repl), [Codesandbox](https://codesandbox.io/s/babel-repl-custom-plugin-7s08o?file=/src/index.js), GitHub Repo, or screenshot if applicable:
```

Comment:
👎 A screenshot is impossible to copy-paste, and it makes things harder for us. It's better is users directly paste their code.

Reply:
I guess I didn't mean it as an alternative but get your point!

Reply:
FWIW, I don't think @hzoo is advocating that screenshot is good, just that it's better than nothing.